packages/foam-core/package.json

```diff
@@ -15,18 +15,23 @@
     "prepare": "tsdx build"
   },
   "devDependencies": {
+    "@types/dashify": "^1.0.0",
     "@types/graphlib": "^2.1.6",
     "@types/lodash": "^4.14.157",
+    "glob": "^7.1.6",
```

Comment:
`glob` needs to be a dependency, not a devDependency

Reply:
@jevakallio fixed this